R/layout.R

```diff
@@ -77,12 +77,15 @@ layout_column_wrap <- function(
   #      to fill the grid row.
   #   2. Allow for heights_equal="-cell", which useful for allowing contents to
   #      shrink but not grow (i.e., default flex behavior).
-  children <- lapply(children, function(x) {
-    bindFillRole(
-      container = TRUE,
-      div(bindFillRole(div(x), container = fillable, item = TRUE))
-    )
-  })
+  children <- dropNulls(children)
+  children <- if (length(children) > 0) {
```

Comment:
Is the `length()` check needed?

Reply:
Not strictly, no. I added it to ensure the `children` is `NULL` when added to the column wrap `div()`, but I hadn't realized that `div(list())` is equivalent to `div()`. There's a bit of implicit behavior either way (implied `NULL` vs implied dropping of empty lists), so I'll take out the length check.